doc/changelog.rst

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+* :feature:`757` Add ``--warnings-as-errors`` flag for ``slash list``
```

Comment:
```slash list tests``` now accepts the ``--warnings-as-errors`` flag, making it treat warnings it encounters as errors